static/src/javascripts/projects/common/modules/experiments/tests/remind-me-email.js

```diff
@@ -0,0 +1,194 @@
+                        dataLinkName: 'one-off_series-description_blind-date'
+                    },
+                    experience: {
+                        subtitle: null,
+                        title: 'What is <b>Experience?</b>',
+                        description: 'Every Saturday a Guardian reader shares their unique story',
+                        cta: 'Remind me next Saturday',
+                        dataLinkName: 'one-off_series-description_experience'
+                    }
+                }
+            },
+            weekly: {
+                comingUp: {
+                    blindDate: {
+                        subtitle: 'Coming up in next week&rsquo;s Blind Date',
+                        title: '<b>&lsquo;She didn’t judge me for talking about giraffes&rsquo; necks&rsquo;</b>',
+                        description: 'Did digital communications consultant Lucy, 23, click with waitress Charlie, 20?',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_coming-up_blind-date'
+                    },
+                    experience: {
+                        subtitle: 'Coming up in next week&rsquo;s Experience',
+                        title: '<b>I was stranded in the wilderness for nine days</b>',
+                        description: '&lsquo;I had lost so much weight that I looked like walking skeleton. I was ready to give up&rsquo;',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_coming-up_experience'
+                    }
+                },
+                seriesDescription: {
+                    blindDate: {
+                        subtitle: null,
+                        title: 'What is <b>Blind Date?</b>',
+                        description: 'Every Saturday we fix up two Guardian readers',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_series-description_blind-date'
+                    },
+                    experience: {
+                        subtitle: null,
+                        title: 'What is <b>Experience?</b>',
+                        description: 'Every Saturday a Guardian reader shares their unique story',
+                        cta: 'Get a weekly reminder',
+                        dataLinkName: 'weekly_series-description_experience'
+                    }
+                }
+            }
+        };
+
+        function isBlindDate() {
+            return window.guardian.config.page.series === 'Blind date';
+        }
+
+        function isExperience() {
+            return window.guardian.config.page.series === 'Experience';
+        }
+
+        function insertSignupBox(templateData) {
+            var params = defaults({
+                svg: svg(envelope, ['submit-input__icon'])
+            }, templateData);
+
+            var $signupBox = $.create(template(remindMeEmailSignup, params));
```

Comment:
I don't think you need to call create on this as you seem to just be inserting it directly into the DOM anyway and can just do that with the returned mark-up?